docs/usermanual/build-ldapext-test-plan.html

```diff
@@ -442,7 +442,10 @@ <h2 id="add_threadbind">8b.3.1 Adding a Thread bind Request<a class="sectionlink
 
 <i>(Optional)</i> Check the box Use Secure LDAP Protocol if you access with LDAP over SSL (ldaps)
 </li>
+<li>
```

Comment:
For information, you don't need to commit HTML, it's generated from xml in xdocs